bin/gwdetchar-scattering

```diff
@@ -497,6 +499,15 @@ for i, channel in enumerate(sorted(alldata[0].keys())):
 # close panel group
 page.div.close()
 
+# write a csv file.                           
```

Comment:
@siddharth101, can you add a log message here? something like

```suggestion
# write a csv file
logger.debug('Writing a summary CSV record')
```

Reply:
Okay.